### DISCUSSION TITLE
Unable to compile (because of triangle==20200424 dependency)

### DISCUSSION THREAD

**Discussion by zdila** [azavea/raster-vision#1668] [General]
Hello.

`rastervision_pytorch_learner` depends on `triangle==20200424` which I am unable to install (`triangle/core.c:19951:5: error: lvalue required as increment operand`). I can install `triangle==20220202`. Is it possible for you to upgrade triangle version denendency?

I am using Debian Bookworm (testing).

**Comment by AdeelH** [comment]
Which version of python are you using? I would recommend using Python 3.9 (ideally Anaconda) for best results.

That said, I have just merged #1580 to upgrade the dependency version. But you might have to wait some days for the next release.

**Reply by zdila** [reply]
Thank you. I am using 3.10.9 (provided by Linux distro). I will try it with 3.9 or wait for the release and will let you know :).

**Reply by AdeelH** [reply]
New Raster Vision release is out now. It uses the newer `triangle` version. (You might still run into trouble with other dependencies if installing on Python 3.10 though).
